eval_results.yaml

```
@@ -0,0 +1,26 @@
+- dataset:
+    id: cais/hle                  # Required. A valid dataset id from the Hub, which should be a Benchmark.
+                                  # ^Basically, where does the leaderboard live.
+    config: {dataset_config}      # Optional. The name of the dataset subset or config (when it has different subsets)
+    split: {dataset_split}        # Optional. Example: test, validation
+    revision: {dataset_revision}  # Optional. Example: 5503434ddd753f426f4b38109466949a1217c2bb
```

in case the tuple (dataset, config, split) is not enough to map to a an eval.yml task, you could add task_id maybe ? cc @NathanHB 

```suggestion
    revision: {dataset_revision}  # Optional. Example: 5503434ddd753f426f4b38109466949a1217c2bb
    task_id: {dataset_revision}  # Optional, in case there are multiple tasks for this dataset, config and split combination. Example: solver_pass_at_10
```

alternatively, only require task_id but personally I like the idea of having this in the general case, without task_id

```yml
- dataset:
    id: Idavidrein/gpqa
    config: gpqa_diamond
  value: 0.412
```

(no strong opinion though)

---

yes